driver/ppm_fillers.c

```diff
@@ -2722,6 +2727,37 @@ static int f_sys_poll_x(struct event_filler_arguments *args)
 	return add_sentinel(args);
 }
 
+#define PPM_MS_MGC_MSK 0xffff0000
+#define PPM_MS_MGC_VAL 0xC0ED0000
+
+static int f_sys_mount_x(struct event_filler_arguments *args)
+{
+	unsigned long val;
+	int res;
+	u32 j;
+
+	/*
+	 * Like { PPM_AUTOFILL, 5, APT_REG, { {AF_ID_RETVAL}, {0}, {1}, {2}, {3} } }
+	 * but fix flags in arg 3.
+	 * See http://lxr.free-electrons.com/source/fs/namespace.c?v=4.2#L2650
+	 */
+	val = syscall_get_return_value(current, args->regs);
+	res = val_to_ring(args, val, 0, false, 0);
+	if (unlikely(res != PPM_SUCCESS))
+		return res;
+
+	for (j = 0; j < 4; j++) {
+		syscall_get_arguments(current, args->regs, j, 1, &val);
+		if (j == 3 && (val & PPM_MS_MGC_MSK) == PPM_MS_MGC_VAL)
+			val &= ~PPM_MS_MGC_MSK;
+		res = val_to_ring(args, val, 0, false, 0);
```

Comment:
Sorry, probably my previous comment was not clear. In this case you have 4 arguments, which the first 3 are user buffers (`char*`) so they need `true` and we put them on exit event. The last is mount flags, which is an integer, so it should use `false`, stay on enter event and apply the mask you did.